src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
```

Comment:
Should this have any aliases? Should this drink maybe show if, say, you search for `Riesling`?

Reply:
Hmm, I don't know. Maybe just the word "Alsatian" or "Alsatian Reisling". I push for a lot of aliases because, to me, IMO, this is the whole appeal of the app. Finding by ingredient, and I think there's a near 0 probability anyone ever types exactly this into the search field. However, if that is what the ingredient is and there are no reasonable search aliases, we shouldn't force any. Up to you.